docs/examples/example_si222.md

```
@@ -106,10 +106,20 @@ easyunfold unfold calculate Si_super_deformed/WAVECAR
+
+
+Install the git-lfs was not installed when you clone the repository, install it via:
+
+```
+sudo apt install git-lfs
+git lfs install
+```
+
+Download the files:
 
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)